health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/components/commentPopUp.js

```
@@ -0,0 +1,49 @@
+        },
+        changeQueryName: businessId,
+    }
+    );
+
+    return (
+        <PopUp
+            style={{ width: "700px" }}
+            onClose={onClose}
+            heading={t(heading)}
+            onOverlayClick={onClose}
+            children={[
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)